bitcoin/src/network/constants.rs

```diff
@@ -173,6 +173,44 @@ impl Network {
     }
 }
 
+#[cfg(feature = "serde")]
+pub mod as_core_arg {
```

Comment:
Hmm, now that I think about it maybe we should name it `network_as_core_arg`. Anyway I also kinda dislike `Network` being in `constants`, so let's leave it for now and maybe move/rename later.